src/lib/components/Navbar.svelte

```diff
@@ -89,86 +88,98 @@ The navbar component.
-                            props: { options: Object.keys(chapters[nextRef.chapter]) }
-                        }
-                    }}
-                    active="Book"
-                    on:menuaction={navigateReference}
-                />
-            </svelte:fragment>
-        </Dropdown>
+        {#if $globalConfig.mainFeatures['book-select'] === 'grid' || $globalConfig.mainFeatures['book-select'] === 'list'}
+            <Dropdown>
+                <svelte:fragment slot="label">
+                    {nextRef.book}
+                    <DropdownIcon />
+                </svelte:fragment>
+                <svelte:fragment slot="content">
+                    <TabsMenu
+                        bind:this={bookSelector}
+                        options={{
+                            Book: {
+                                component: SelectGrid,
+                                /**
+                                 * TODO:
+                                 * - add book abbreviations to catalog to be used in UI instead of bookCode
+                                 */
+                                props: {
+                                    options: books.map((b) => b.bookCode) /*bookAbbreviations*/
+                                }
+                            },
+                            Chapter: {
+                                component: SelectGrid,
+                                props: { options: Object.keys(chapters) }
+                            },
+                            Verse: {
+                                component: SelectGrid,
+                                props: { options: Object.keys(chapters[nextRef.chapter]) }
+                            }
+                        }}
+                        active="Book"
+                        on:menuaction={navigateReference}
+                    />
+                </svelte:fragment>
+            </Dropdown>
+        {/if}
         <!-- Chapter Selector -->
-        <Dropdown>
-            <svelte:fragment slot="label">
-                {nextRef.chapter}
-                <DropdownIcon />
-            </svelte:fragment>
-            <svelte:fragment slot="content">
-                <TabsMenu
-                    bind:this={chapterSelector}
-                    options={{
-                        Chapter: {
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters) }
-                        },
-                        Verse: {
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters[nextRef.chapter]) }
-                        }
-                    }}
-                    active="Chapter"
-                    on:menuaction={navigateReference}
-                />
-            </svelte:fragment>
-        </Dropdown>
+        {#if $globalConfig.mainFeatures['show-chapter-number-on-app-bar']}
```

Comment:
Is there a separate issue to implement 'show-chapter-selector' and 'show-chapter-selector-after-book'?

Reply:
Yes, checked in with Jake he said there was